ngirimana/murikahomeltd
# #173244296  add login form

> #### What does this PR do?
* add login form
> #### Description of Task to be completed
- add login form
- Add spinner
- Add heading
- add links
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **ft-login-api-integration-173244296**  branch by running ```git checkoutft-login-api-integration-173244296``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the login link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173244296](https://www.pivotaltracker.com/story/show/173244296)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38111965/84277851-51f6cf80-ab34-11ea-8172-837a2cd168b8.png)
